sortedList/sortedList/list.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <stdbool.h>
+
+struct SortedList; //pointer to the first element of the list
```

Comment:
Комментарии пишутся над комментируемой сущностью, не справа. И отделяются пробелом от символа комментария.
```suggestion
// Pointer to the first element of the list.
struct SortedList;
```